xml/signals/Reading-1954/aspects.xml

```diff
@@ -5,7 +5,7 @@
     xmlns:xsi = "http://www.w3.org/2001/XMLSchema-instance"
     xsi:noNamespaceSchemaLocation = "http://jmri.org/xml/schema/aspecttable.xsd"
 >
-  <name>Reading 1954</name>
+  <name>Reading</name>
   <date>1956</date>
```

Comment:
Question: What happens if there is another Reading, with an earlier or later date? Maybe it's good to retain the date in the name, but I don't know enough of signal systems to have an opinion about that. Just checking.

Reply:
One possible answer:  It depends.  If the rules published one year give an _identical_ JMRI signaling system implementation to a signaling system implementation for rules published in a different year, then "no harm, no foul".  If, however, the rules change in a way that requires changes to the signaling system, then that may or may not be problematic.  If aspects change between rules-sets from different years, then that's justification for a separate JMRI signaling system definition.  If the difference is that a new style of signal mast was added, then I'd think it could simply be an "update" to the existing JMRI signaling system, although users might be confused by the "anachronism" implied by "newer mast types" in a signaling system based on rules for an older date.